src/condor_startd.V6/VolumeManager.cpp

```diff
@@ -255,7 +262,7 @@ isHideMountCompatible(const ClassAd& ad) {
 
 
 bool
-VolumeManager::CheckHideMount(ClassAd* ad, bool& hide_mount) {
+VolumeManager::CheckHideMount(ClassAd* jobAd, ClassAd* machineAd, bool& hide_mount) {
```

Comment:
Can we make the `ClassAd *machineAd` const?

And if it can't ever be a `nullptr`, can it be a const ClassAd & instead of a pointer?

Reply:
Discussed in person:
- The ClassAd pointers should definitely be `const`
- Changing to references is a bit to much of a change, and I prefer having the LVM code doing the check for the ads  due to the ads presence (or lack there of) having different effects depending on the hide mount configuration which I don't want to leave up to the outside caller of the function to get correct.